nerdtree_plugin/fs_menu.vim

```diff
@@ -203,10 +203,22 @@ endfunction
 function! NERDTreeListNode()
     let treenode = g:NERDTreeFileNode.GetSelected()
     if treenode != {}
-        let metadata = split(system('ls -ld ' . shellescape(treenode.path.str())), '\n')
+        if has("osx")
+            let stat_cmd = 'stat -f "%z" '
+        else
+            let stat_cmd = 'stat -c "%s" '
+        endif
+
+        let cmd = 'size=$(' . stat_cmd . shellescape(treenode.path.str()) . ') && ' .
+        \         'size_with_commas=$(echo $size | sed -e :a -e "s/\(.*[0-9]\)\([0-9]\{3\}\)/\1,\2/;ta") && ' .
+        \         'ls -ld ' . shellescape(treenode.path.str()) . ' | sed -e "s/ $size / $size_with_commas /" && ' .
+        \         'unset size && ' .
```

Comment:
Do you need to unset `size` and `size_with_commas` here?  Note that this command is run within a subshell.

Reply:
Nope. Those are leftover from my tweaking the command line in the shell.